js/carousel.js

```diff
@@ -1,3 +1,68 @@
+const slides = [
+  {
+    image: "img/movie_photo/Spider-man.png",
+    title: "Spider-Man: No Way Home (2021)",
+    genre: "Action,Adventure,Fantasy • 2h 28m",
+    about: "With Spider-Man's identity now revealed, Peter asks Doctor Strange for help. When a spell goes wrong, dangerous foes from other worlds start to appear, forcing Peter to discover what it truly means to be Spider-Man.",
+    smallAbout: "With Spider-Man's identity now revealed, Peter asks Doctor Strange for help. When a spell goes wrong, dangerous...",
+    smallAbout2: "With Spider-Man's identity now revealed, Peter asks Doctor Strange for help. When a spell..."
+  },
+  {
+    image: "img/movie_photo/hotel.png",
+    title: "Tansylvania",
+    genre: "Animation,Adventure,Comedy • 1h 27m",
+    about: "After one experiment, Johnny turns into a monster and everyone else becomes human. Now it has to be seen whether they will be able to reverse this experiment.",
+    smallAbout: "After one experiment, Johnny turns into a monster and everyone else becomes human. Now it has to be seen whether they...",
+    smallAbout2: "After one experiment, Johnny turns into a monster and everyone else becomes human. Now it has to..."
+  },
+  {
+    image: "img/movie_photo/avatar.png",
+    title: "AVATAR 2: The Way of Water (2022)",
+    genre: "Action,Adventure,Sci-Fi • 2h 10m",
+    about: "Jake Sully lives with his newfound family formed on the planet of Pandora. Once a familiar threat returns to finish what was previously started, Jake must work with Neytiri and the army of the Na'vi race to protect their planet.",
+    smallAbout: "Jake Sully lives with his newfound family formed on the planet of Pandora. Once a familiar threat returns to finish what was...",
+    smallAbout2: "Jake Sully lives with his newfound family formed on the planet of Pandora. Once a familiar threat..."
+  }
+]
+
+
+function movie(URL,movieName,genre,about,sAbout,sAbout2) {
```

Comment:
well, for now, it's okay, but the next time try to avoid this type of element generation, it can cause some problems.